client/packages/portal-ui/src/lib/styleProvider/styleProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import { EdsProvider, Icon } from '@equinor/eds-core-react';
+import * as icons from '@equinor/eds-icons';
+import { GlobalColorStyle } from './globalColors';
+import { GlobalStyle } from './gobalStyles';
+
+Icon.add(icons);

```

Comment:
consider adding the icons you use

Reply:
will a add coment to do that and make a list of used ones ;)